app/matplotlib.ipynb

```diff
@@ -233,22 +357,47 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
-   "outputs": [],
    "source": [
     "x = np.linspace(0, 5, 100)\n",
     "y1 = np.sin(x)\n",
     "y2 = np.cos(x)\n",
     "\n",
-    "# write your code here\n"
+    "fig, (ax1, ax2) = plt.subplots(1, 2, figsize=(12, 6))\n",
+    "\n",
+    "ax1.plot(x, y1, color='blue')\n",
+    "ax1.set_title('Sine Function')\n",
+    "ax1.set_xlabel('x')\n",
+    "ax1.set_ylabel('sin(x)')\n",
+    "\n",
+    "ax2.plot(x, y2, color='green')\n",
+    "ax2.set_title('Cosine Function')\n",
+    "ax2.set_xlabel('x')\n",
+    "ax2.set_ylabel('cos(x)')\n",
+    "\n",
+    "fig.suptitle('Sine and Cosine Functions', fontsize=16)\n",
+    "\n",
+    "plt.tight_layout()\n"
```

Comment:
The subplot is missing the command to display the plot. You should add `plt.show()` at the end to display the plot.